src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { fetchBooks } from './components/redux/books/books';
+import BookList from './components/BookList';
+import Navigation from './components/NavBar';
+import CategoryPage from './components/Category';
+
+const App = () => {
+  const dispatch = useDispatch();
+  dispatch(fetchBooks());
+  return (
+    <div>
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<BookList />} />
```

Comment:
- [x] You are required to have a  `<Books>` component, which contains the following components: `<BookList>` which renders a list of books using the `<Book>` component. Following this structure, it is the `Books` component that should be imported into the highlighted piece of code. 👍

Reply:
fixed